### DISCUSSION TITLE
Feet vs Fathoms

### DISCUSSION THREAD

**Discussion by drreynholds** [jaybo/deepzoom-blog#28] [Q&A] [answered]
Is there a way to show depth in feet rather than fathoms?

**Comment by jaybo** [comment]
DeepZoom currently uses raster charts.  Each individual chart uses either feet, fathoms, fathoms and feet, or (rarely) meters.  A given geographic area might have five different charts for the various zoom levels, each with its own particular units.

DeepZoom displays the units for the current chart at the current zoom level for the location _at the center of the screen_ in the upper left hand corner.

![image](https://github.com/jaybo/deepzoom-blog/assets/1607323/14cbe938-8401-4d39-9b55-f0cd6a3af1af)

Vector charts don't have these kind of limitations.  But they also lack the lore and artistry of raster charts.

**Comment by jaybo** [comment, marked as answer]
Note that as of v3.14, all USA charts show depths in feet.
